<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea" type="bluesea_node" output="screen" >
    <param name="type" value="uart" />
    <param name="port" value="/dev/ttyUSB0" />
    <param name="frame_id" value="map" />
    <param name="baud_rate" value="230400" />
    <param name="output_scan" value="1" />
    <param name="output_cloud" value="0"/>
    <param name="output_360" value="1"/>   
    <param name="mirror" value="1"/>
    <param name="from_zero" value="1"/>
    <param name="with_resample" value="true"/>
    <param name="resample_res" value="0.6"/>
    <param name="angle_patch" value="0"/>
    <param name="unit_is_mm" value="0"/>
    <param name="with_confidence" value="1"/>
    <param name="with_checksum" value="0"/>  
  </node>
 </group>
</launch>
